website/recursion_website/templates/base.html

```diff
@@ -0,0 +1,59 @@
+<!Doctype html>
+<html>
+<head>
+	<title>RECursion</title>
+	<!--Package Imports-->
+
+	<!-- Compiled and minified MATERIAL CSS -->
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/materialize/1.0.0/css/materialize.min.css">
+    <!-- Compiled and minified MATERIAL JavaScript -->
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/materialize/1.0.0/js/materialize.min.js"></script>
+
+    <link rel="stylesheet" href="{% static 'css/base_css.css' %}">
+</head>
+<body>
+	<header>
+	<nav><!-- Navigation Color is controlled by class !-->
+		<div class="nav-wrapper #263238 blue-grey darken-3">
+			<a href="#" class="brand-logo"><img class="img" src="http://www.recursionnitd.in/images/log.png"></a>
+			<ul id="nav-mobile" class="right hide-on-med-and-down">
+				<li><a href="/events.html">Events</a></li>
+				<li><a href="/askREC.html">AskREC</a></li>
+				<li><a href="/start.html">Getting Started</a></li>

```

Comment:
Name of the html file needs to be changed, "start" does not signify anything. Better to rename it "gettingStartedPage" or something related.